.github/workflows/test-inputs.yml

```diff
@@ -429,3 +429,115 @@ jobs:
         uses: ./action/
         with:
           release-type: major
+
+  test-baseline-version:
+    # WIP: Help
```

Comment:
Perhaps leftover from an earlier iteration?

Reply:
Yep, leftover from the previous PR. Fixed.